src/start_location.cpp

```diff
@@ -491,11 +491,11 @@ void start_location::handle_heli_crash( avatar &you ) const
             // Damage + Bleed
             case 1:
             case 2:
-                you.add_effect( effect_bleed, 6_minutes, bp );
+                you.add_effect( effect_bleed, 4_minutes * roll, bp );
```

Comment:
This change is for fun and not necessary for the rest of the PR. The average bleed time is still 6 minutes 1/4 of the time.